package.json

```
@@ -4,8 +4,9 @@
     "start": "ts-node src/index.ts",
     "lint": "prettier --check --write --parser typescript \"{src,test}/**/*.ts\"",
     "lint:fail": "prettier --check --parser typescript \"{src,test}/**/*.ts\"",
-    "jest": "jest",
-    "test": "yarn lint && yarn jest"
+    "vitest": "vitest",
+    "test": "yarn lint && vitest",
+    "test:ui": "yarn lint && vitest --ui"
```

How about adding coverage check using official [`@vitest/coverage-c8`](https://www.npmjs.com/package/@vitest/coverage-c8) package?

Just in case, `@vitest/coverage-c8` has to be installed as a dev dependencies.

```suggestion
    "vitest": "vitest --coverage",
    "test": "yarn lint && vitest --coverage",
    "test:ui": "yarn lint && vitest --coverage --ui"
```